scripts/demo.py

```diff
@@ -45,13 +45,15 @@ def main(args):
     frames_or_path = prepare_frames_or_path(args.video_path)
     prompts = load_txt(args.txt_path)
 
+    frame_rate = 30
     if args.save_to_video:
         if osp.isdir(args.video_path):
-            frames = sorted([osp.join(args.video_path, f) for f in os.listdir(args.video_path) if f.endswith(".jpg")])
+            frames = sorted([osp.join(args.video_path, f) for f in os.listdir(args.video_path) if f.endswith((".jpg", ".jpeg", ".JPG", ".JPEG"))])
```

Comment:
I guess the syntax would be better if you do something like this

```
frames = sorted([
osp.join(args.video_path, f) 
for f in os.listdir(args.video_path) 
if (extension := f.split(".")[-1].lower()) in ("jpg", "jpeg")
])
```

And yeah, why to limit only JPEG extension?